### ISSUE TITLE
Publish Parent POM to simplify keeping Exporters in Sync

### ISSUE THREAD

**Issue by qqmyers** [IQSS/dataverse#10482]
**Overview of the Feature Request** 
Several [recent automated PRs in the dataverse-exporters repo](https://github.com/gdcc/dataverse-exporters/pulls) are to update libraries that are also in use in Dataverse. It would be useful to be able to keep those in sync with the version in Dataverse, which, per @poikilotherm , would be easier if the Dataverse parent pom is published to maven central. We discussed today that this might be something that could be looked into around ~June 2024 (If anyone has time before then feel free to pick this up!) with work to update the example and current external exporters to adopt using the Dataverse parent pom being done ~ in parallel. Having the pom available from maven could help other projects as well and may also allow exporters to be smaller (and use the libraries already deployed in the war).

**Any brand new behavior do you want to add to Dataverse?**
This is nominally a change in the repo rather than the Dataverse software per se.

**Any open or closed issues related to this feature request?**

**Comment by poikilotherm** [comment]
I started looking into this and began by creating a universal parent POM at GDCC we can base on, so we don't need to maintain all of this stuff in a whole lot of places.

https://github.com/gdcc/maven-parent

This will make the Dataverse Parent much shorter.

**Comment by cmbz** [comment]
2024/07/19 - 6.4 proposal request from @poikilotherm

**Comment by cmbz** [comment]
2024/08/28 - @poikilotherm are you planning to get back to this work within the next two weeks? If so we can move forward for 6.4. Otherwise we'll have to wait for 6.5.

**Comment by pdurbin** [comment]
I removed the 6.4 milestone after [checking](https://dataverse.zulipchat.com/#narrow/stream/379673-dev/topic/universal.20parent.20pom/near/471024759) with @poikilotherm about it.

**Comment by cmbz** [comment]
2024/10/10: moved to sprint ready at request of @pdurbin now that it has been sized.

**Comment by cmbz** [comment]
2024/11/07: Placing it On Hold temporarily until Oliver works on it and creates a Pull Request.
